http/src/main/resources/leo.conf

```diff
@@ -165,6 +165,17 @@ azure {
         token-scope = ${?AZURE_MI_TOKEN_SCOPE}
         token-acquisition-timeout = ${?AZURE_MI_TOKEN_ACQUISITION_TIMEOUT}
       }
+      publisher-config {
+        topic-name = ${?AZURE_SB_TOPIC_NAME}
```

Comment:
hmm... where do these values live? I would have expected to find them [here](https://github.com/broadinstitute/terra-helmfile/blob/master/values/app/leonardo/live.yaml.gotmpl#L215)

Reply:
These values are only relevant when deploying on Azure, for the current deployment on GCP these values should not be set. On Azure, Leo will use Azure Service Bus instead of GCP pub/sub as the messaging back end.